Client/Ecosia/Extensions/BrowserViewController+Ecosia.swift

```diff
@@ -3,8 +3,9 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 import UIKit
-import Core
+
```

Comment:
```suggestion
```

This is also the case that there is a space between imports in a number of other files. Is there any specific reason for it?

Reply:
Yeah, that was due to a bundled filter->rename action. No specific reason for that.

Reply:
Shall we get rid of all those spaces then?